modules/cert-manager-acme-dns01-ambient-aws.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-ambient-aws_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Amazon Route 53.
+
+[NOTE]
+====
+Private DNS zones are not supported.
+====
```

Comment:
Can you move this note in the `security/cert_manager_operator/cert-manager-acme-about.adoc` file? Then you don't have to add it in each section repeatedly.